core/downloadfromdcthread.h

```diff
@@ -33,6 +33,7 @@ class DCDeviceData {
 	bool forceDownload() const;
 	bool saveLog() const;
 	int diveId() const;
+	bool syncTime() const;
```

Comment:
Not a comment to this PR, but are these getter/setter pairs really necessary?

Reply:
It could probably be done in a better way, but this will need some refactoring of the mobile app.